openhands/integrations/github/templates/pr_update_prompt.j2

```diff
@@ -0,0 +1,25 @@
+You are working in /workspace, checked out to branch {{ branch_name }}, which has an open PR #{{ pr_number }}.
```

Comment:
we should remove these references to `/workspace`. The runtime is CD'ed into the repo dir, and this ends up confusing the agent